packages/wrangler/src/autoconfig/frameworks/utils/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./vite-config";
```

Comment:
nit: Personally I don't think we should be using barrel exports like this, especially for internal imports